### PR TITLE
Search reorganization: search installation overview

### PR DESCRIPTION
**Pull request by Alec-Shay**
Originally from @justinchoi001 at: https://github.com/jrhoun/liferay-learn/pull/92

The directory for this article in the original PR is out-of-date, so here it is moved into where our tentative outline has it (in the `DevOps` section for search). Also, I renamed the article to "Introduction to Installing a Search Engine," since it doesn't actually give steps for installing them (instead just introducing the supported search engines and comparing the installations).

There is quite a bit of information here about the limitations of Solr, which maybe could belong better in a Solr-specific article? Since we don't have one here yet though, I've left it still as part of this article.

### CODE REVIEW

**Comment by rbohl** [comment]
Thanks Alec. 
We wouldn't want to move the Solr limitations out of the intro, because someone trying to decide which search engine to use needs to see that information. Though I suppose we could more briefly mention that solr has limitations, and link to a dedicated article on it in the Solr section.

FYI, Solr our Solr connector will see one more release (Solr 8 IIRC) and then we're dropping it.

**Comment by Alec-Shay** [comment]
For sure, even if that information does get separated out from the intro, there should at the very least be mention of those limitations in the intro (and then linking to a dedicated article or section), since it does bring up Solr as a topic.

File: docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/introduction-to-installing-a-search-engine.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+### Elasticsearch and Java Distribution Compatibility
+
+Consult the [Elasticsearch compatibility matrix](https://www.elastic.co/support/matrix#matrix_jvm) and the [Liferay DXP compatibility matrix](https://help.liferay.com/hc/sections/360002103292-Compatibility-Matrix) to learn more about supported JDK distributions and versions.
+
+### Solr and Java Distribution Compatibility
+
+This consideration is not necessary for Solr, because no JVM level serialization happens between the servers. All communication occurs at the HTTP level.
+
+## Configuring a Search Engine
+
```
Comment (jrhoun) [inline docs/dxp/7.x/en/using-search/installing-and-upgrading-a-search-engine/introduction-to-installing-a-search-engine.md:24]: Assuming @Alec-Shay this section was intentionally left blank?

**Comment by jrhoun** [comment]
Merged. Thanks!

**Pull request closed without merging**